databricks/koalas/plot.py

```diff
@@ -1118,6 +1118,73 @@ def _plot(data, x=None, y=None, subplots=False, ax=None, kind="line", **kwds):
     return plot_obj.result
 
 
+_backends = {}
+import importlib
+
+
+def _find_backend(backend: str):
+    """
+    Find a pandas plotting backend
+    """
+    # function copied from pandas.plotting._core
+
+    import pkg_resources  # Delay import for performance.
+
+    for entry_point in pkg_resources.iter_entry_points("koalas_plotting_backends"):
+        if entry_point.name == "matplotlib":
+            # matplotlib is an optional dependency. When
+            # missing, this would raise.
+            continue
+        _backends[entry_point.name] = entry_point.load()
+
+    try:
+        return _backends[backend]
+    except KeyError:
+        try:
+            module = importlib.import_module(backend)
+        except ImportError:
+            # We re-raise later on.
+            pass
+        else:
+            if hasattr(module, "plot"):
+                # Validate that the interface is implemented when the option
+                # is set, rather than at plot time.
+                _backends[backend] = module
+                return module
+
+    raise ValueError(
+        f"Could not find plotting backend '{backend}'. Ensure that you've installed ",
+        f"the package providing the '{backend}' entrypoint, or that the package has a ",
+        "top-level `.plot` method.",
```

Comment:
The f-strings is not supported in Python 3.5. Could you use `"...".format(...)` instead?